content/en/news/releases/1.6.x/announcing-1.6/_index.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Announcing Istio 1.6
+linktitle: 1.6
+subtitle: Major Update
+description: Istio 1.6 release announcement.
+publishdate: 2020-05-19
+release: 1.6.0
+skip_list: true
+aliases:
+    - /news/announcing-1.6.0
+    - /news/announcing-1.6
+---
+
+We are pleased to announce the release of Istio 1.6!
+
+{{< relnote >}}
+
+With this release, we are continuing the path we charted earlier this year in
+our [roadmap post](/blog/2020/tradewinds-2020/), sailing toward more
+simplicity, a better installation experience, and some other goodies as well.
+
+Here’s some of what’s coming to you in today's release:
+
+## Simplify, simplify, simplify
+
+Last release, we introduced **Istiod**, a component that reduced the number of
+components in an Istio installation by combining the functionality of several
+other components. In Istio 1.6, you'll find that Citadel, the sidecar
+injector and Galley are no longer deployed.
+
+You'll also find that we are using the new `AppProtocol` API from Kubernetes.
```

Comment:
`appProtocol` is not an API, rather an internal field in the the EndpointPort API.

A better way to write this sentence would be
```
"You'll also find that we are using the new `appProtocol` field in the [EndpointPort](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#endpoints-v1-core) API.```

Reply:
great suggestion. done.

Reply:
nit: Kubernetes users don't know what EndpointPort is, its an internal name. We can just clarify that its for selecting the protocol in Service. And maybe link to https://preliminary.istio.io/docs/ops/configuration/traffic-management/protocol-selection/#manual-protocol-selection instead of k8s api where we explain how and why to use this